decidim-admin/app/models/decidim/admin/abilities/participatory_process_admin_user.rb

```diff
@@ -9,55 +9,45 @@ module Abilities
       # Admin engine.
       #
       # This ability will not apply to organization admins.
-      class ParticipatoryProcessAdmin
-        include CanCan::Ability
-
-        def initialize(user, _context)
-          @user = user
-
-          return unless user && !user.role?(:admin) && !user.role?(:collaborator)
-
-          can :read, :admin_dashboard do
-            participatory_processes.any?
-          end
+      class ParticipatoryProcessAdminUser < Decidim::Abilities::ParticipatoryProcessAdminUser
```

Comment:
Well.. I personally dislike the `User` suffix but it looks like a convention we followed in the past and I just normalised it.
We could use `Ability` like: `CurrentUserAbility`, `EveryoneAbility`, `ParticipatoryProcessAdminAbility` and so on.
Is it fine for you? @josepjaume @deivid-rodriguez